src/libraries/restore/runtime/runtime.depproj

```diff
@@ -98,7 +103,11 @@
           DependsOnTargets="ResolveRuntimeFilesFromLocalBuild"
           AfterTargets="AfterResolveReferences;FilterNugetPackages"
           Condition="'$(RuntimeFlavor)' == 'Mono'">
-    <ItemGroup Condition="'$(BinPlaceTestRuntimePack)' != 'true'">
+    <ItemGroup>
```

Comment:
Once you do what I suggested here: https://github.com/dotnet/runtime/pull/35295/files#r416829953 You can remove this `ItemGroup` and then move the `TargetsMobile != true` condition to the target itself.

`Condition="'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' != 'true'"`

Reply:
thanks!

Reply:
I think I might have just broken the tests with this suggestion :smile:. Because you're adding `RuntimeFiles` to `ReferenceCopyLocalPaths` and then removing them in `SetupTestingHost`. Let's instead change how these targets are hooked up. 

Add `DependsOnTargets=OverrideRuntimeMono` to `SetupTestingHost` target and bring back what you had.

Reply:
Ah okay will change this now. I was wondering if it was breaking because I forgot the `'$(RuntimeFlavor)' == 'Mono'` check. 
Just wondering, what in the build errors in CI would indicate its caused by changes in this PR?

Reply:
Because the mono tests are crashing because it can’t find some runtime files and your first commit was green.

Reply:
Something still isn't right.  libcoreclr is ending up in lib.

Reply:
Actually.... `libcoreclr.dylib` was being placed into `lib-runtime-packs/runtimes/<rid>/lib/netcoreapp5.0/` when the build was all green too. I didn't catch it prior. Checking right now if mobile also includes that.

Reply:
I think the whole `ReferenceLocalCopyPaths` clearing caused the problem.  I threw up a commit where we BinPlace the native libs + runtime into testhost.  That basically piggybacks off of what we already have done.